pkg/core/pipeline/pullRequests.go

```diff
@@ -74,12 +74,6 @@ func (p *Pipeline) RunPullRequests() error {
 
 		failedTargetIDs, attentionTargetIDs, successTargetIDs, skippedTargetIDs := p.GetTargetsIDByResult(pr.Config.Targets)
```

Comment:
```suggestion
		failedTargetIDs, _, successTargetIDs, skippedTargetIDs := p.GetTargetsIDByResult(pr.Config.Targets)
```
Is `attentionTargetIDs` still needed if we remove it in the next section?

Reply:
Yes, it is needed in https://github.com/updatecli/updatecli/pull/427/files#diff-1f1229b2326f3c175da892cc978cc23d3c23bdf7485c59328b6fabc7288e65b1R89 for the loop that retrieves the changelogs.

Don't worry, this kind of situation (e.g. "variable not used") is always caught by both the linter and the compiler :)

Reply:
In fact this change should have been in https://github.com/updatecli/updatecli/pull/426/files

Reply:
Not really, because #425 took care of this and was merged first